src/components/input/input.scss

```
@@ -181,6 +181,7 @@ md-input-container {
     -ms-flex-preferred-size: $input-line-height; //IE fix
     border-radius: 0;
     border-style: solid; // Firefox fix
+    transition: border-color 0.4s cubic-bezier(.25,.8,.25,1);
```

```suggestion
    transition: border-color $swift-ease-out-duration $swift-ease-out-timing-function;
```
We've got some variables defined already. They can be used to make this a bit easier.